src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -78,7 +82,13 @@ def set_color(self, color):
         pass
 
     def set_background_color(self, color):
-        pass
+        if color is None:
+            self.native.backgroundColor = NSColor.windowBackgroundColor
+        elif color is TRANSPARENT:
+            self.native.backgroundColor = NSColor.clearColor
+            self.native.drawsBackground = False
```

Comment:
If you set color to TRANSPARENT, and then back to any other color - doesn't this mean that `drawsBackground` will remain False?

Reply:
Good catch, thanks!